### ISSUE TITLE
How to set base url for image tag

### ISSUE THREAD

**Issue by huy-lv** [jsdf/react-native-htmlview#304]
Hello, i have an image tag like this, how to set base url for all `src` attributes.
```
<img class='img-responsive'  src='/images/article/image-1598870719460.jpg'>
```

**Comment by dinhsuu** [comment]
is it like this ?
`<HTML
          baseFontStyle={styles.textContent}
          html={section.content}
          ignoredStyles={['font-family']}
          renderers={{
            img: (img) => {
              return (
                <FastImage
                  key={img}
                  style={{ width: 300 * UIConst.SCALE, height: (img.height / 2) * UIConst.SCALE }}
                  source={{ uri: `https://aaaaa.com${img.src}` }}
                  resizeMode={'contain'}
                />
              );
            }
          }}
        />

>`

**Comment by huy-lv** [comment]
Yes, but i need a prop help me to set base url for all image tag, something like this
```
<HTML html={html} baseUrl={"https://a.com"}/>
```
